sippy-ng/src/releases/PayloadStreamOverview.js

```diff
@@ -0,0 +1,170 @@
+import {
+  Card,
+  Container,
+  Grid,
+  makeStyles,
+  Tooltip,
+  Typography,
+} from '@material-ui/core'
+import { DataGrid } from '@material-ui/data-grid'
+import { Error } from '@material-ui/icons'
+import { safeEncodeURIComponent, SafeJSONParam } from '../helpers'
+import { StringParam, useQueryParam } from 'use-query-params'
+import { TEST_THRESHOLDS } from '../constants'
+import Alert from '@material-ui/lab/Alert'
+import GridToolbar from '../datagrid/GridToolbar'
+import InfoIcon from '@material-ui/icons/Info'
+import PassRateIcon from '../components/PassRateIcon'
+import PropTypes from 'prop-types'
+import React, { Fragment, useEffect } from 'react'
+import SummaryCard from '../components/SummaryCard'
+
+function PayloadStreamOverview(props) {
+  const [release = props.release, setRelease] = useQueryParam(
+    'release',
+    StringParam
+  )
+  const [arch = props.arch, setArch] = useQueryParam('arch', StringParam)
+  const [stream = props.stream, setStream] = useQueryParam(
+    'stream',
+    StringParam
+  )
+
+  const [fetchError, setFetchError] = React.useState('')
+  const [isLoaded, setLoaded] = React.useState(false)
+  const [streamHealth, setStreamHealth] = React.useState([])
+
+  const fetchData = () => {
+    let queryString = ''
+
+    if (props.release !== '') {
+      queryString += '&release=' + safeEncodeURIComponent(props.release)
+    }
+
+    fetch(
+      process.env.REACT_APP_API_URL +
+        '/api/releases/health?' +
+        queryString.substring(1)
+    )
+      .then((response) => {
+        if (response.status !== 200) {
+          throw new Error('server returned ' + response.status)
+        }
+        return response.json()
+      })
+      .then((json) => {
+        // Find our specific stream's health in the list:
+        for (const stream of json) {
+          if (
+            stream.architecture == props.arch &&
+            stream.stream == props.stream
```

Comment:
`===` not `==` is generally preferred for this kind of comparison (https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/)

Reply:
TIL, thanks!